dependencymagnet/doc.go

```diff
@@ -5,8 +5,5 @@
 package dependencymagnet
 
 import (
-	_ "github.com/jteeuwen/go-bindata/go-bindata"
-	_ "github.com/openshift/library-go/alpha-build-machinery"
-	_ "k8s.io/code-generator"
 	_ "github.com/openshift/api/samples"
```

Comment:
@gabemontero is this needed to fetch the vendored CRD YAML files?

Reply:
yes @adambkaplan 

with the src code now importing `github.com/openshift/api/samples` this entire file is technically unnecessary

but this magnet file will be useful next time we deal with the chicken/egg scenario where we want to
- go mod vendor a new dependency
- then have your favorite golang editor find that dependency when you update your code to use the new dependency

so I left the one entry we still use

but if you feel strongly about it I can just delete the dependency magnet file

Reply:
I'm going to go ahead and remove this file altogether @adambkaplan 

I've learned `go mod` is better/easier with golang 1.13 about bringing in new dependencies since originally crafting this PR.  This file is now "fully" unneeded IMO :-)

Will update shortly.